---
layout: archive
title: "Deciphering the trophic strategies of microbial eukaryotes"
permalink: /research/
author_profile: true
---

### **Single-celled microbial eukaryotes / protists / microeuks**
What microbial eukaryotes are present in a marine environment? and what are they doing? These are the motivating questions of my research. Protistan (or microbial eukaryote) species make up the majority of the branches of the eukaryotic tree of life with their diverse physical forms (morphologies) and metabolisms or trophic strategies.   


### **Ecology of microbial eukaryotes at hydrothermal vents**
Highly reduced and thermally charged venting fluids discharged from the subseafloor mix with surrounding seawater at hydrothermal vents â€“ this creates a sharp geochemical gradient which promotes a hub of biological diversity. While studies of prokaryotic diversity at hydrothermal vent sites have highlighted the important roles microorganisms play in deep sea carbon cycling and offered a unique window into subseafloor microbial communities, depictions of deep-sea marine ecology and food webs are incomplete without characterization of single-celled microbial eukaryotes (protists). I am currently working on culture-independent techniques (tag-sequencing and metatranscriptomics) to provide a thorough understanding of protistan biogeography in and near venting fluids, focusing on the vent fluid-seawater interface. 
  I am specifically focusing on the role of phagotrophic protists at these niche environments. By coupling grazing and RNA-SIP experiments we can hone in on this important trophic interaction. 
Quantifying the role of heterotrophic protists is incredibly important, as these processes form the foundation of deep-sea marine food webs and may mediate a significant amount of carbon transferred to higher trophic levels.


### **Shifting metabolic priorities observed among ecologically important protists**

To gain a better understanding of the ecological activities of marine protists, we can sequence the collective mRNA of all eukaryotes in the community - metatranscriptomics. Metatranscriptomics allows us to characterize both the taxonomic and functional diversity. The availability of improved genetic databases for microbial eukaryotes in recent years has enabled researchers to apply transcriptomic approaches to natural samples; however this technique is still infrequently applied in the field due to the limited number of reference transcriptomes (and subsequent annotations) in sequence databases. Comparative metatranscriptomic surveys indicated shifts in metabolic potential (changes in transcript abundance) with respect to taxonomic group and environmental factors, providing a transcript-level metric of the microbial functional diversity underscoring important biological processes. These efforts are transforming the way biologists and ecologists describe microbial populations, as incorporation of functional traits into predictive ecosystem models (e.g. phytoplankton functional type, PFT) enables clearer representation of ocean ecosystems. At the San Pedro Ocean Time-series station, comparisons of relative transcript abundances revealed depth-related shifts in the nutritional modes of key protistan taxonomic groups. These can be linked to nutritional strategies such as phototrophy, heterotrophy, or mixotrophy. Functional annotations from sunlit samples were typical of phytoplankton communities (i.e. glycolysis, Calvin cycle, and fatty acid synthesis), while taxa capable of phagotrophy below the euphotic zone were characterized by an upregulation of transcripts associated with the breakdown of fatty acids and the glyoxylate cycle. Additionally, we found evidence for anaerobic metabolism among ciliates, and adaptations to prolonged darkness in diatoms and chlorophytes. We show that separating group-specific nutritional modes is a novel approach for examining linkages between protistan community composition and ecosystem function (i.e. microbially mediated biological processes).
[Hu et al. 2018](https://onlinelibrary.wiley.com/doi/abs/10.1111/1462-2920.14259)

### **Coupled RNA- and DNA-based tag-sequencing to infer potential activity**

One challenge in characterizing microbial eukaryote diversity using high-throughput tag-sequencing data is that most studies rely on rDNA from the environment, which does not discriminate the genetic material from live, active microorganisms from non-viable, non-living genetic material. Ribosomal RNA (rRNA) is more susceptible to degradation than rDNA, thus sequence information derived from rRNA provides an estimate of ribosomal content and thus a proxy for protein synthesis in a cell. By pairing traditional rDNA tag-sequencing with sequences derived from rRNA (cDNA reverse transcribed from extracted total RNA). By using RNA-derived tag-sequences, we can be more confident that we are more likely capturing the active protistan community. Using this approach we showed how the community composition and metabolic activity of dominant protistan taxa fluctuated with respect to season and location. Samples from the San Pedro Channel, off the coast of California, were collected seasonally at three environmentally distinct sites: the San Pedro Ocean Time-series (SPOT) station from four depths (surface, deep chlorophyll maximum, 150 m, and 890 m), surface waters offshore from Santa Catalina Island, and from the Port of Los Angeles. rRNA estimates of community diversity reflected seasonal changes in community structure, while rDNA-based estimates did not. Temporal changes in the RNA:DNA ratios of regionally important phytoplankton corresponded to known bloom activity, thus validating the utility of combined rRNA and rDNA sequencing methods in efforts to monitor phytoplankton blooms. Previous rDNA-based work at SPOT detected ciliates and radiolaria below the euphotic zone (150 m and 890 m), but was unable to confirm if sequences originated from live cells or dead sinking material. Our results supported the presence of metabolically active ciliate and radiolarian groups at 150 m and 890 m, demonstrating how comparisons of RNA:DNA ratios were useful for gathering ecological information from species that are otherwise difficult to characterize [(Hu et al. 2016)](https://academic.oup.com/femsec/article/92/4/fiw050/2197988).   

Microbially-mediated biological processes are known to synchronize with regular light-dark (diel) cycles, but short-term temporal dynamics influencing protistan community structure are largely undocumented. A Lagrangian survey in the oligotrophic North Pacific Subtropical Gyre (NPSG) was conducted every 4 hours over a period of 3 days to investigate diel rhythmicity in the relative abundances and activities of euphotic zone protists. The relative metabolic activity (inferred from RNA:DNA ratios) of several phytoplankton groups was found to coordinate with the light cycle; the exact timing of daily peaks in RNA:DNA ratios varied with respect to group, which reflected the varied phytoplankton metabolic strategies (e.g. biomass-specific nutrient uptake rates or timing of photosynthetic activity). Further, the relative metabolic activities of phagotrophic protists were more similar to one another consistently peaking at dusk and throughout the dark cycle, suggesting that grazing activity increased in response to availability of prey. These results implicated temporal niche partitioning within the protistan community, influenced by light-dark cycling. Time-dependent, significantly co-occurring OTUs (as a proxy for species) throughout the diel cycle were hypothesized to represent important predator-prey, parasitic, and mutualistic relationships. More specifically, negative or positive temporally time-delayed interactions with the parasitoid Syndiniales included a high number of dinoflagellates and ciliates, suggesting Syndiniales parasitism may contribute more to mortality in the NPSG than previously thought. Further, the frequency of mutualistic relationships between endosymbiotic algae and heterotrophic rhizarian hosts (e.g. acantharia) were inferred from positively co-correlated OTUs. These findings are unique, as traditional molecular surveys do not typically include these symbiotic trophic interactions [(Hu et al. 2018)](https://www.frontiersin.org/articles/10.3389/fmars.2018.00351/abstract).

### **Protistan diversity using high-throughput sequencing**

_The V4 hypervariable region_

Advances in DNA sequencing capabilities over the past 20 years have enhanced studies of microbial ecology by providing an additional metric for testing phylogenetic hypotheses, supplementing classic microscopy-based approaches for species identification, and uncovering vast amounts of previously unknown diversity in virtually all environments on the planet. A major challenge in using DNA sequence information to catalog protistan species composition is the process of accurately assigning taxonomic identities and drawing ecological interpretations from sequence results. The most common approach for taxonomy characterization of a diverse community is to perform high-throughput tag-sequencing of a selected hypervariable region, which falls within the highly conserved 18S rRNA gene. However, limitations in high-throughput sequence read lengths [currently, maximum lengths generated from Illumina paired end sequencing are ~400-500 bps] do not provide the same taxonomic resolution as longer sequences (e.g. the entire 18S rRNA gene). Ecological intepretations from various hypervariable regions (of differing lengths) may differ when compared to results derived from full length 18S rRNA gene sequences [(Hu et al. 2015)](https://onlinelibrary.wiley.com/doi/abs/10.1111/jeu.12217). Currently, methods to apply full-length tag sequencing of conserved regions (like 18S) are not practical at large scales (i.e., too costly) - but targetting the longer V4 region (~400bp) where there are curated reference databases (like PR2) is a suitable alternative for characterizing diverse _in situ_ microbial populations.

![Amplicons](/images/amplicon.png)

_Opinions and moving forward_

The exponential increase in sequence-based analyses of protistan species richness raises the question: 'Have we moved beyond characterizing species-level diversity?' and begun to reveal intraspecies diversity. This is likely true for some protistan lineages, but if we are to move forward with documenting _microdiversity_, it must be justified and protistologists should recognize and acknowledge this in their analyses. This is specifically relevant for applying Operational Taxonomic Units (OTUs) or Amplicon Sequence Variants (ASVs) - intraspecies variability can be an important component of protistan physiology and behavior. [Caron & Hu 2019](https://www.sciencedirect.com/science/article/abs/pii/S0966842X18302324).

***

{% if author.googlescholar %}
  You can also find my articles on <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>
{% endif %}

{% include base_path %}

{% for post in site.publications reversed %}
  {% include archive-single.html %}
{% endfor %}

